test/images/agnhost/README.md

```diff
@@ -17,18 +17,8 @@ cases for the same tested behaviour.
 
 ## Usage
 
-The `agnhost` binary is a CLI with the following subcommands:
-
-- `dns-suffix`: It will output the host's configured DNS suffix list, separated by commas.
-- `dns-server-list`: It will output the host's configured DNS servers, separated by commas.
-- `etc-hosts`: It will output the contents of host's `hosts` file. This file's location
-  is `/etc/hosts` on Linux, while on Windows it is `C:/Windows/System32/drivers/etc/hosts`.
-- `pause`: It will pause the execution of the binary. This can be used for containers
-  which have to be kept in a `Running` state for various purposes, including executing
-  other `agnhost` commands.
-- `help`: Prints the binary's help menu. Additionally, it can be followed by another
-  subcommand in order to get more information about that subcommand, including its
-  possible arguments.
+The `agnhost` binary has several subcommands which are can be used to test different
```

Comment:
I'm not really sure what this `agnhost` means but question is - is this name still relevant after adding so many other subcommands?

Reply:
Eh, less so. So, the name ``agnhost`` comes from "agnhost host", which basically means that it'll work the same on any OS. But it is still true, the image still contains the common Linux utilities on both Linux and Windows, including /bin/sh, nc, etc., and 3 of the current subcommands still accomplish the original goal. Adding the other subcommands doesn't necessarely go against its goal.